test/models/Stafftest.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file Defines tests for the Student sailsjs model.
+ */
+
+"use strict";
+const helperIntegrationTests = require("./_helpers");
+
+describe("Staff model", function() {
+    let testData = {
+        associations: {},
+        record: {}
+    };
+
+    let testStaff = {
+        username: "TESTSTUDENT@DEWV.EDU",
```

Comment:
Minor detail really, but you gave the staff a student email address.

Reply:
@Emreyildirim1284 We also have a convention of using DEWV.NET instead of DEWV.EDU in test data.